Bottom: e7a21cebf2fa327fb4f5b5cde9d848217e20201e
Top:    add3d09abf65a17c5b271456018f2b97426f5bf4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 21:59:02 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/draw.c b/src/draw.c
index 9932d97..ddad0a2 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -414,8 +414,6 @@ DrawEverything (BoxTypePtr drawn_area)
   /* This is the reverse of the order in which we draw them.  */
   int drawn_groups[MAX_LAYER];
 
-  printf ("WARNING\n");
-
   PCB->Data->SILKLAYER.Color = PCB->ElementColor;
   PCB->Data->BACKSILKLAYER.Color = PCB->InvisibleObjectsColor;
